libs/openchallenges/challenge/src/lib/challenge-routing.module.ts

```diff
@@ -1,13 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { ChallengeComponent } from './challenge.component';
-import { ChallengeOverviewModule } from './challenge-overview/challenge-overview.module';
 
-const routes: Routes = [
+export const routes: Routes = [
+  { path: '', component: ChallengeComponent },
   {
-    path: ':challengeName',
+    path: ':challengeSlug',
```

Comment:
@rrchai Just to clarify, we will use the challenge ID to identify a challenge, not its slug (see Google doc).

Reply:
It's actually the second route parameter (optional)